### PR TITLE
Jjdzr14 ba 14

### PR DESCRIPTION
**Pull request by szablewski**

### CODE REVIEW

**Review by szablewski — COMMENTED** [review]
Kilka uwag w komentarzach

**Review by piotrek-piotrek-dev — APPROVED** [review]
aside from comments, all looks good

File: .idea/.gitignore
Comment (piotrek-piotrek-dev) [inline .idea/.gitignore:1]: do we need this file?

File: .idea/jjdzr14-Bankersi2k24.iml
Comment (piotrek-piotrek-dev) [inline .idea/jjdzr14-Bankersi2k24.iml:1]: do we need this file?

File: .idea/misc.xml
Comment (piotrek-piotrek-dev) [inline .idea/misc.xml:1]: do we need this file?

File: .idea/vcs.xml
Comment (piotrek-piotrek-dev) [inline .idea/vcs.xml:1]: do we need this file?

**Review by piotrek-piotrek-dev — COMMENTED** [review]
do we need those files in .idea dir?

**Pull request merged**